### DISCUSSION TITLE
Recognition faults - sometimes

### DISCUSSION THREAD

**Discussion by dersupergrobi** [jomjol/AI-on-the-edge-device#1251] [Q&A]
Hi,

sometime my recognition doesnt work properly, although it worked before.
trying to attach a pic later…from my Watermeter that shows 00726 with digitals, recognition is 8726 (i dont use the first 0), but it worked perfect

Any idea how 2 fix?

Frank

...![image](https://user-images.githubusercontent.com/16088406/199194741-8812968b-00ce-46e4-939b-44bdbca425ea.jpeg)

**Comment by jomjol** [comment]
Please follow the recommended settings for the ROIs (size, position), then the recognition rate will get much better (see wiki)
